sci-libs/nfft/nfft-3.5.2.ebuild

```diff
@@ -34,16 +34,19 @@ pkg_pretend() {
 
 src_configure() {
 	econf \
-	    --enable-all \
+		--enable-all \
 		--enable-shared \
-		$(use_enable openmp) \
-		$(use_enable static-libs static)
+		--disable-static \
+		$(use_enable openmp)
 }
 
 src_install() {
 	default
-	use doc || rm -r "${ED}/usr/share/doc/${P}/html" || die
 
-	# infft uses pkg-config to record its private dependencies
+	if use !doc; then
```

Comment:
prefer bash negation over use negation, i.e., to `if ! use doc; then`

Reply:
Done